build/DotNetSdkVersions.txt

```diff
@@ -2,3 +2,4 @@
 -Channel 3.1 -Runtime dotnet
 -Channel 8.0 -Runtime dotnet
 -Channel 9.0.3xx
+-Channel 10.0.1xx -Quality daily
```

Comment:
Can you elaborate why?

We don't target .NET 10 in this repo, so we shouldn't need the 10.0 SDK.